recipes/faiss/conda_build_config.yaml

```diff
@@ -0,0 +1,18 @@
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- None
+- 9.2                               # [linux]
+- 10.0                              # [linux]
+- 10.1                              # [linux]
+- 10.2                              # [linux]
+docker_image:
+- an_irrelevant_value               # [not linux]
+- condaforge/linux-anvil-comp7      # [linux]
+- condaforge/linux-anvil-cuda:9.2   # [linux]
+- condaforge/linux-anvil-cuda:10.0  # [linux]
+- condaforge/linux-anvil-cuda:10.1  # [linux]
+- condaforge/linux-anvil-cuda:10.2  # [linux]
+zip_keys:
+  - - cuda_compiler_version
+    - docker_image
```

Comment:
We can drop this file. Conda-Smithy will add this for us based on `conda-forge-pinning` when re-rendering the feedstock.

Reply:
OK fair enough, will try drop it again. It's just that osx/windows didn't seem to know the `cuda_compiler_version` variable without this file (and ucx-split is linux-only, so this wrinkle doesn't appear there).

Reply:
Good point. I'll make some suggestions so we can address that 🙂

Reply:
@h-vetinari, can you remove this file?

Reply:
As mentioned, I tried this with https://github.com/conda-forge/staged-recipes/pull/11337/commits/92380101af50280e6a28d927190551ff2f9a5fc5, and it [ends up](https://github.com/conda-forge/staged-recipes/pull/11337#issuecomment-626076199) skipping the osx builds - this time because it's stumbling over `skip: true # [cuda_compiler_version != "None"]`, but before I squash-cleaned up the commits a bit, I definitely had some failing CI runs on osx because `cuda_compiler_version` wasn't recognised on osx without this file.

Reply:
Please have a look at https://github.com/conda-forge/arrow-cpp-feedstock/blob/0.16.x/recipe/meta.yaml#L46 for how to do it correctly.